docs/data/joy/getting-started/tutorial/tutorial.md

```diff
@@ -89,9 +89,13 @@ The [Typography](/joy-ui/react-typography/) component replaces HTML header, para
 
 :::info
```

Comment:
I think we can remove this content from the info callout ⎯ seems ok to be in the page's body!